### PR TITLE
Add Dutch language support

### LINKED ISSUE #126: Updated translations for 'nl': Dutch (Nederlands, Vlaams)
Your translation have been copied to your clipboard. Paste it here:

Updated translations for 'nl': Dutch (Nederlands, Vlaams):
```json
{
	"disAll": {
		"message": "Alles uitschakelen",
		"hash": "a9f66d43cfde31fd3cd61c3627eec653"
	},
	"enableAll": {
		"message": "Alles inschakelen",
		"hash": "bff1a5f49fdf6aef0eff0315299dea83"
	},
	"extDesc": {
		"message": "Bekijk, schakel in en uit of verwijder je extensie met gemak.",
		"hash": "557ac2dd0ce9b4249f7dc37868bcfccf"
	},
	"extName": {
		"message": "One Click Extensions Manager",
		"hash": "9f7e3f659272f7b34a4073ddd8e29658"
	},
	"gotoOpt": {
		"message": "Opties…",
		"hash": "0912d504999e2cb4f65b37eddb39b7a7"
	},
	"hideInfoMsg": {
		"message": "Verberg dit",
		"hash": "814f376f8a44776d5b80020e1327d173"
	},
	"manage": {
		"message": "Extensie pagina…",
		"hash": "a93cc3eb11188cfa36bc2639e0f5ea89"
	},
	"openUrl": {
		"message": "Bekijk op de Web Store",
		"hash": "54c9abfc9842db7d4a2c0b692d15140c"
	},
	"searchTxt": {
		"message": "Zoek op naam…",
		"hash": "1220702106584c8c90c0f41c486a3e0b"
	},
	"undoInfoMsg": {
		"message": "Je kan je laatste actie ongedaan maken of opnieuw uitvoeren totdat je deze popup sluit. Druk op <kbd>ctrl+z</kbd> op Windows of Ubuntu,  en <kbd>cmd+z</kbd> op MacOS.",
		"hash": "b8be6c1fc88c7e6042b054effb2c6981"
	},
	"uninstall": {
		"message": "Verwijderen uit Chrome",
		"hash": "a04a561e0b5c8b0210a17382781cabd1"
	},

	"__WET_LOCALE__": { "message": "nl" }
}

```

### PR DESCRIPTION
**Pull request by sommmen**
Closes #126

### CODE REVIEW

File: source/_locales/nl/messages.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	"gotoOpt": {
+		"message": "Opties…",
+		"hash": "0912d504999e2cb4f65b37eddb39b7a7"
+	},
+	"hideInfoMsg": {
+		"message": "Verberg dit",
+		"hash": "814f376f8a44776d5b80020e1327d173"
+	},
+	"manage": {
+		"message": "Extensie pagina…",
```
Comment (fregante) [inline source/_locales/nl/messages.json:27]: Thank you! Can you make sure that this matches  the wording that Chrome uses in this menu?


Screenshot

**Review by fregante — APPROVED** [review]

File: source/_locales/nl/messages.json (hunk 1 above)
Comment (sommmen) [reply source/_locales/nl/messages.json:27]: I'm not sure what you mean exactly i think you want me to set the 'manage' variable to either:

Extensions -> Extensies
Manage extensions -> Extensies beheren

Which one would you like? can i edit the file as-is (given there is a 'hash' value) or should i go through the translator tool again?

Comment (fregante) [reply source/_locales/nl/messages.json:27]: I'm not sure what the hash is for, I don't see it on Chrome's docs. You can go ahead and manually update the file here without the hash

Comment (sommmen) [reply source/_locales/nl/messages.json:27]: @fregante the hash was added by https://lusito.github.io/web-ext-translator/?gh=https://github.com/hankxdev/one-click-extensions-manager/tree/main

I think its just for the tool or something.

Anyways :

Extensions -> Extensies
Manage extensions -> Extensies beheren

Which one would you like?

Comment (fregante) [reply source/_locales/nl/messages.json:27]: Oh forgot to mention. The second one

**Comment by fregante** [comment]
Prettier needs to be run locally unfortunately

**Comment by sommmen** [comment]
> Prettier needs to be run locally unfortunately

Is that something you could pick up at some point? i don't have the toolchain - i'm mainly doing c#.
Also annoying that it doesn't show what rules are violated but okay :).

**Comment by hankxdev** [comment]
@fregante able to merge?

**Pull request merged**